x-pack/plugins/apm/public/components/routing/templates/no_data_config.ts

```diff
@@ -48,13 +48,9 @@ export function getNoDataConfig({
     solution: i18n.translate('xpack.apm.noDataConfig.solutionName', {
       defaultMessage: 'Observability',
     }),
-    actions: {
+    action: {
       elasticAgent: {
         title: noDataConfigDetails.title,
-        description: i18n.translate('xpack.apm.ux.overview.agent.description', {
-          defaultMessage:
-            'Use APM agents to collect APM data. We make it easy with agents for many popular languages.',
-        }),
```

Comment:
@alisonelizabeth This was definitely an oversight from the conversion to the new shared_ux package. @majagrubic We need to support these custom descriptions. Can you fix this?

Reply:
Thank you! I was wondering about that 🤔. @majagrubic let me know if I can help in anyway with this.

Reply:
I can help out here on Monday.  @cchaos and @alisonelizabeth I feel like this PR can ship as-is and we can follow up with the fix?  If so, I would file an issue and include it in the source code here... I'd hate to block this, but if it's going to be in review for a few more days, we can get the fix first.  Just let me know.

Reply:
I just published https://github.com/elastic/kibana/pull/133594 which will address the issue.

Reply:
Thanks @clintandrewhall! I'm OK to merge as-is for now. I'd also prefer not to block on this.

Reply:
Well, if we can get that fix PR merged, you can just rebase and be golden.  Hopefully someone on my team can review this morning.